### PR TITLE
[quantization] Add left/right padded static prefill runtime

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds left/right padded static prefill runtime for llama.

Related: https://github.com/Samsung/TICO/pull/660#discussion_r3151989416, #586
TICO-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

```bash
python tico/quantization/wrapq/examples/static_llama_layer_runtime.py --model ~/hd/model/Llama-3.2-1B-Instruct --padding-side left
====================================================================================================
Step 0: prefill last-token logits
padding_side = left
prefill input shape = (1, 256)
valid lengths = [6]
mean|diff| = 1.99095643
 max|diff| = 11.44373512
PEIR    = 42.931876 %
----------------------------------------------------------------------------------------------------
Step 1: decode logits
sequence lengths = [7]
mean|diff| = 1.39612782
 max|diff| = 11.07236576
PEIR       = 38.278456 %
----------------------------------------------------------------------------------------------------
Step 2: decode logits
sequence lengths = [8]
mean|diff| = 1.45166409
 max|diff| = 10.16841602
PEIR       = 43.110922 %
----------------------------------------------------------------------------------------------------
Step 3: decode logits
sequence lengths = [9]
mean|diff| = 2.97339845
 max|diff| = 16.40535927
PEIR       = 69.787032 %
----------------------------------------------------------------------------------------------------
Step 4: decode logits
sequence lengths = [10]
mean|diff| = 3.89460278
 max|diff| = 13.47235870
PEIR       = 39.967973 %
----------------------------------------------------------------------------------------------------
Step 5: decode logits
sequence lengths = [11]
mean|diff| = 3.83032703
 max|diff| = 17.50397682
PEIR       = 40.907990 %
----------------------------------------------------------------------------------------------------
Step 6: decode logits
sequence lengths = [12]
mean|diff| = 4.03509808
 max|diff| = 18.72264671
PEIR       = 51.972480 %
====================================================================================================
Verification finished.
====================================================================================================
Generated text:
The capital of France is Paris. The Eiffel Tower is located in Paris. The Louvre Museum
```

### CODE REVIEW

**Review by stamalakhov — APPROVED** [review]
LGTM!

**Comment by stamalakhov** [comment]
Compared to https://github.com/Samsung/TICO/pull/587 PEIR is high though. Padding is not for free (just my personal opinion).

**Comment by mhs4670go** [comment]
> Compared to #587 PEIR is high though. Padding is not for free (just my personal opinion).

Ah, I found that this is caused from k_proj fusion. I'll post a patch.
